### PR TITLE
Onboarding UI Components

### PR DESCRIPTION
**Pull request by aml3ed**
Adds:
- Slideshow component for use with onboarding flow
- Barebones tag component for use within following tags (visual not functional)
Changes:
- The way SparkCards are displayed to give context to what a spark is and also make it look less like a post
- OrgForm now directly integrates logic for profile picture upload rather than all of that existing in the app.  Now app just has to pass the imageUploadHandler and it's good to go
- PostForm and OrgForm now have additional props for button displaying

### CODE REVIEW

File: src/components/Slideshow/Slideshow.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        previousSlide: {
+          pathname: location.pathname,
+          hash: location.hash,
+          search: queryString.stringify({
+            ...queryString.parse(location.search),
+            [name]: parseInt(queryString.parse(location.search)[name]) - 1 || 0
+          })
+        }
+      })}
+      <div className={classNames(className, style["slide-controller"])}>
```
Comment (kstolte) [inline src/components/Slideshow/Slideshow.js:34]: I think this needs a data-testId here.

File: src/forms/OrgForm/OrgForm.js [hunk 2]
```diff
@@ -14,30 +27,119 @@ const orgSchema = yup.object().shape({
   bio: yup.string()
 });
 
-const OrgForm = ({ onSubmit, defaultValues = {}, validate = false }) => (
+// eslint-disable-next-line react/prop-types
+const AvatarEdit = ({ initialURL, imageUploadHandler = () => {} }) => {
+  const fileRef = useRef();
+  const [avatarSrc, setSrc] = useState(initialURL);
+  const { notifyDispatch = () => console.log("Notify") } = useNotify();
```
Comment (kstolte) [inline src/forms/OrgForm/OrgForm.js:34]: is this intended?

File: src/forms/PostForm/PostForm.js [hunk 3]
```diff
@@ -86,7 +87,7 @@ const PostForm = ({
         defaultValue={parseContent(defaultValues?.content)}
         reset={reset}
         name="content"
-        placeholder="What's the latest..."
+        placeholder="What's on you mind..."
```
Comment (kstolte) [inline src/forms/PostForm/PostForm.js:90]:
```suggestion
        placeholder="What's on your mind..."
```

File: src/templates/Tag/Tag.js [hunk 4]
```diff
@@ -1,21 +1,66 @@
...
+        backgroundImage: `linear-gradient(rgba(0, 0, 0, 0.50), rgba(0, 0, 0, 0.50)), url(${tag?.backgroundImage ||
+          "https://cdn.lumastic.com/media/v1/pages/demo/Header.png"})`
+      }}
+    >
+      <div className={style.push}>
+        <Link inline to={tagRoute(tag?.name)} className={style.name}>
+          <Signature>
+            <Hashtag />
+            <Type headerFont>
+              <b>{tag.name || "woodworking"}</b>
```
Comment (kstolte) [inline src/templates/Tag/Tag.js:26]: lols default is woodworking.

**Review by kstolte — CHANGES_REQUESTED** [review]

File: src/templates/Tag/Tag.js (hunk 4 above)
Comment (aml3ed) [reply src/templates/Tag/Tag.js:26]: Hahaha yup

File: src/components/Slideshow/Slideshow.js (hunk 1 above)
Comment (aml3ed) [reply src/components/Slideshow/Slideshow.js:34]: Yup you're super right

File: src/forms/OrgForm/OrgForm.js (hunk 2 above)
Comment (aml3ed) [reply src/forms/OrgForm/OrgForm.js:34]: Yes - It needs to be own component so it can update form state

**Review by kstolte — APPROVED** [review]

**Pull request merged**